pkg/api/handlers/compat/images.go

```diff
@@ -192,9 +205,22 @@ func CreateImageFromSrc(w http.ResponseWriter, r *http.Request) {
 			utils.Error(w, "Something went wrong.", http.StatusInternalServerError, errors.Wrap(err, "failed to write temporary file"))
 		}
 	}
+
```

Comment:
```
 	opts := entities.ImageImportOptions{}
 	platformSpecs := strings.Split(query.Platform, "/")
	opts.OS = platformSpecs[0]
        if len(platformSpecs) > 1 {
    		opts.Architecture = platformSpecs[1]
        }
```

Reply:
right, this makes sense. Will change now